### PR TITLE
[FCE-259] Remove ios example

### PR DESCRIPTION
**Pull request by karkakol**
## Description

Remove iOS example and files related to it.

### CODE REVIEW

File: packages/ios-client/project.json [hunk 1]
```diff
@@ -0,0 +1,222 @@
+{
```
Comment (graszka22) [inline packages/ios-client/project.json:1]: is this file needed?

Comment (karkakol) [reply packages/ios-client/project.json:1]: This package is created by script `gen-mocks.sh`. It is necessary for iOS tests. Not sure though is necessary in repo. On the other hand it affects testing soo i think it should be in repo. If you think the opposite give me your thoughts.

Comment (karkakol) [reply packages/ios-client/project.json]: Never mind. Deleted.

**Review by graszka22 — APPROVED** [review]
